src/main.ts

```diff
@@ -0,0 +1,4 @@
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
```

Comment:
Let's only have aot files. There's no need to demonstrate JIT compilation in this repo, it's just distracting.